src/Symfony/Component/Security/Tests/Acl/Voter/AclVoterTest.php

```diff
@@ -27,7 +27,7 @@ class AclVoterTest extends \PHPUnit_Framework_TestCase
      */
     public function testSupportsAttribute($attribute, $supported)
     {
-        list($voter, , $permissionMap, ,) = $this->getVoter(true, false);
+        list($voter, , $permissionMap, , ) = $this->getVoter(true, false);
```

Comment:
That commas indicate size of array on the right side - how many data we can catch into list statement.
I can see how this can be helpful, but I don't use it too.
Remove on whole branch using https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1028